Core/Core/DocViewer/PSPDFAnnotationExtension.swift

```diff
@@ -58,12 +58,11 @@ extension PSPDFAnnotation {
             annotation = strikeout
         case .freetext:
             let freeText = PSPDFFreeTextAnnotation(contents: apiAnnotation.contents ?? "")
-            let fontInfo = apiAnnotation.font?.split(separator: " ")
             let fontSizeStr = apiAnnotation.font?.trimmingCharacters(in: CharacterSet(charactersIn: "0123456789").inverted) ?? ""
-            freeText.fontName = fontInfo?.last.flatMap { String($0) } ?? "Verdana"
-            freeText.fontSize = CGFloat(Int(fontSizeStr) ?? 14)
-            freeText.fillColor = .white
-            freeText.sizeToFit()
```

Comment:
I know this was added to fix some really bad text clipping. It's possible PSPDFKit has fixed things though. I do wonder if it's safe to keep around though? Not blocking, want to know what you think.

Reply:
We set the bounding box further down. Whatever size this determines is thrown away.